packages/data-stores/src/reader/mutations/use-subscription-manager-site-delivery-frequency-mutation.ts

```diff
@@ -0,0 +1,69 @@
+import { translate } from 'i18n-calypso';
+import { useMutation, useQueryClient } from 'react-query';
+import { callApi } from '../helpers';
+import { useIsLoggedIn } from '../hooks';
+import type { SiteSubscriptionDeliveryFrequency } from '../types';
+
+type SiteSubscriptionDeliveryFrequencyParams = {
+	delivery_frequency: SiteSubscriptionDeliveryFrequency;
+	blog_id: number | string;
+};
+
+type SubscriptionResponse = {
+	blog_ID: string;
+	delivery_frequency: SiteSubscriptionDeliveryFrequency;
+	status: string;
+	ts: Date;
+};
+
+type SiteSubscriptionDeliveryFrequencyResponse = {
+	success: boolean;
+	subscribed: boolean;
+	subscription: SubscriptionResponse | null;
+};
+
+const useSubscriptionManagerSiteDeliveryFrequencyMutation = () => {
+	const isLoggedIn = useIsLoggedIn();
+	const queryClient = useQueryClient();
+	return useMutation<
+		SiteSubscriptionDeliveryFrequencyResponse,
```

Comment:
Not an issue at all, but it is not necessary to explicitly define the types here. They will be inferred. It was probably complaining when it was work in progress, and you did not have well typed params. 🙂

Reply:
You're absolutely right. It was nagging, I added them, but it seems like they are not needed anymore 👍 
I've removed them.